config/config.yaml

```diff
@@ -41,6 +41,15 @@ assets:
         rtl:
             basePath: %webroot%/public/themes/
             link: rtl_%theme_tabs_layout%
+    patientportal-base:
+        basePath: %webroot%/public/themes/portal/
+        link: base.css
+    patientportal-registration:
+        basePath: %webroot%/public/themes/portal/
+        link: register.css
+    patientportal-style:
+        basePath: %webroot%/public/themes/portal/
+        link: style.css
```

Comment:
@sjpadgett , there's a very good chance that these may not be loading right since I don't think build will place these there (if not, prob makes more sense to retitle the script to portal-base.css, portal-register.css, portal-style.css and move them to interface/themes; then the build process will copy them to public/themes automatically)

Reply:
This was @tywrenn fixin's:)

Reply:
Yeah never got a chance to test these and I just realized they're broken 😂

Reply:
no prob, I'll bring in changes tonight to fix this

Reply:
I just brought in the fixes for this into the PR